pkg/services/secrets/kvstore/migrations/migrator.go

```diff
@@ -78,6 +78,10 @@ func (s *SecretMigrationProviderImpl) Migrate(ctx context.Context) error {
 			logger.Debug("Finished secret migration service", "service", serviceName)
 		}
 	})
+	if err != nil {
+		logger.Warn("Server lock for secret migration already exists")
```

Comment:
I think this should still be a logger.Error

Reply:
Sounds good! Will do!

Reply:
Done on 526bc81318